docs/source/development/quickstart.rst

```diff
@@ -5,9 +5,10 @@ Quick start
 ================================================================================
 
 This is a short introduction to the PROJ API. In the following section we
-create a simple program that transforms a geodetic coordinate to UTM and back
-again. The program is explained a few lines at a time. The complete program can
-be seen at the end of the section.
+create two simple programs that illustrate how to transform points between
+two different coordinate systems, and how to convert between projected and
+geodetic coordinates for a single coordinate system. Explanations for individual
```

Comment:
The second example, using a pipeline with `proj_create()` transforms between ED50/UTM32 and ETRS89/UTM33 and not between geodetic and projected coordinates. See exercise from the tutorial for more https://proj.org/tutorials/EUREF2019/exercises/pipelines.html#ed50-utm32-etrs89-utm33

The text here should reflect that.